pkg/applicationserver/io/pubsub/grpc_pubsub.go

```diff
@@ -69,7 +69,12 @@ func (ps *PubSub) List(ctx context.Context, req *ttnpb.ListApplicationPubSubsReq
 
 // Set implements ttnpb.ApplicationPubSubRegistryServer.
 func (ps *PubSub) Set(ctx context.Context, req *ttnpb.SetApplicationPubSubRequest) (*ttnpb.ApplicationPubSub, error) {
-	if err := rights.RequireApplication(ctx, req.ApplicationIdentifiers, ttnpb.RIGHT_APPLICATION_TRAFFIC_DOWN_WRITE); err != nil {
+	if err := rights.RequireApplication(ctx, req.ApplicationIdentifiers,
+		ttnpb.RIGHT_APPLICATION_SETTINGS_BASIC,
+		ttnpb.RIGHT_APPLICATION_TRAFFIC_READ,
+		ttnpb.RIGHT_APPLICATION_TRAFFIC_UP_WRITE,
```

Comment:
Writing uplink is not something that we'll do through pub/sub

Reply:
Removed in 820d316319f8f80da2b92fe6577d72f7e06a554d . Had to rebase in order to get the CI to pass again since `protoc` got updated.